deliverables/README.md

```diff
@@ -12,16 +12,37 @@ Sign Up for Cloud9 and create a basic Rails project.
 ####Required
 
 What is an "interpreted" language?<br>
+ An interpreted language is one where instructions are performed directly without the need for a complier. 
+ 
 What is IRB?<br>
+ IRB is interactive ruby shell. It is used as a Read-Evaluate-Print-Loop (REPL) which is a way to interact and test Ruby programs immediately through the command line. 
+ 
 What are Objects?<br>
+ Objects are instances of classes which means they are created from by calling the correct class in Ruby. Most things in Ruby are objectst. 
+ 
 What are Methods?<br>
+ Methods are the instructions or abalities an object has. It can be rewritten so multiple objects have the same instructions. You can either use the built in ones or create your own. 
+ 
 What are Classes?<br>
+ Classes are the layout or blueprint of objects. 
+ 
 What are Blocks?<br>
+ Blocks are storeable code that can be ran on demand. They are created by using braces in the code or using a do...end statement. 
+ 
 What is an Array?<br>
+ An array is a list of ordered objects. 
+ 
 What is an Iterator?<br>
+ An interator is a command that repreatidly is executed, like a loop. 
+ 
 What are hashes?<br>
+ Hashes are similar to arrays, but instead of an index, hashes use valued keys. 
+ 
 What is a library?<br>
+ The compilation of all of the source code available to accomplish tasks. 
```

Comment:
A little imprecise.  A "library" is just a set of classes and/or functions that are pre-written.  You can import them into your project and use them.